### PR TITLE
WEBUI-927: add functional tests for unselect some for publish action

### PR DESCRIPTION
**Pull request by Pranit-Sotre**

### CODE REVIEW

**Review by rakeshkumar1019 — CHANGES_REQUESTED** [review]
Please investigate why the functional test is failing.

**Review by swarnadipa-dev — DISMISSED** [review]
Looks good

File: ftest/features/publication.feature [hunk 1]
```diff
@@ -133,3 +133,28 @@ Feature: Internal Publication
...
+      | Note       | PublishNote1     | memo    | art/culture             | europe/France        | SJones  | /default-domain/PublishTest     |                  | urgent | sample.odt |
+    And I have permission ReadWrite for the document with path "/default-domain/PublishTest"
+    And I have permission ReadWrite for the document with path "/default-domain/sections/section1"
+    When I browse to the document with path "/default-domain/PublishTest"
+    And I select all the documents
+    And I deselect the "PublishFile" document
+    Then I can publish selection to "section1"
+    When I browse to the document with path "/default-domain/sections/section1"
+    Then I can see the document has 3 children
+    Then I click the toast notification dismiss button
```
Comment (Nisha-QA) [inline ftest/features/publication.feature]: You can delete the line as this is causing intermittent failure in functional PR checks.

**Review by GuruprasadAnk1996 — APPROVED** [review]

**Review by rakeshkumar1019 — APPROVED** [review]

**Review by rajatgoel-gl — APPROVED** [review]

**Review by Nisha-QA — APPROVED** [review]
Looks good

**Review by Gabez0r — APPROVED** [review]

**Pull request merged**